Header header
string interface
bool enabled
bool connected
string ssid
float32 frequency # GHz
string access_point
float32 bitrate # Mb/s
int16 tx_power # dBm
float32 link_quality
int16 signal_level # dBm
